README.md

```diff
@@ -22,11 +22,13 @@ In order to use this extension, please install the following components:
 2. Install the ["Prusti Assistant" extension](https://marketplace.visualstudio.com/items?itemName=viper-admin.prusti-assistant) in Visual Studio Code.
 3. Open a Rust file to activate the extension. At its first activation, this extension will automatically download Prusti and install the required Rust toolchain.
 
-To verify a Rust program, open the command palette (View -> Command Palette, or Shift+Ctrl+P on Ubuntu) and run the command `Prusti: save and verify this file`. You should see a "Running Prusti..." message in the status bar while Prusti is running. When Prusti terminates the result of the verification is reported in the status bar and in the "Problems" tab (open it with View -> Problems).
+**To verify a standalone Rust program**, open the command palette (View -> Command Palette, or Shift+Ctrl+P on Ubuntu) and run the command `Prusti: verify the current file or crate`. You should see a "Running Prusti..." message in the status bar while Prusti is running. When Prusti terminates the result of the verification is reported in the status bar and in the "Problems" tab (open it with View -> Problems).
+
+**To verify crates** instead of standalone programs, enable the corresponding flag in the settings (Preferences -> Settings -> type "Prusti" -> choose "CurrentCrate" from "verificationMode").
```

Comment:
Is there no way to detect if the opened project is a crate? Can we have two commands in the IDE rather than the one configurable one?